queries.js

```diff
@@ -385,7 +385,7 @@ async function findReportrecord_parameters(req, res, next) {
   }
   let report;
   try {
-    if (req.query.hasOwnProperty('report')) {
+    if (req.query.hasOwnProperty('report_id')) {
```

Comment:
I've been using this endpoint with `report` in Shesmu and it's been working. I'm really not sure  how. :confused:

Reply:
Aha, that's why. I couldn't figure out how Shesmu was using that endpoint because I was looking in Bitbucket, not in the Shesmu project. Will revert the endpoint back to `/reportdb/record_start?report_id=$report_id` . @morgantaschuk  I'll make a PR into the Guanyin Perl lib with this reversion. If these API updates are acceptable to everyone after the reversion, I'd like to do a Guanyin release.

Reply:
Ah, you haven't been using this endpoint -- this is the "find report record given user-supplied parameters". Shesmu uses the "create minimal report record" endpoint, which still does need to be fixed. Patch coming soon.

Reply:
It looks like I'm mistaken -- Shesmu _also_ uses the `/reportdb/record_parameters?report=X` endpoint. I'll revert this change after all.